lib/classes/TradeOffer.js

```diff
@@ -477,11 +474,13 @@ TradeOffer.prototype.send = function(callback) {
 	}, (err, response, body) => {
```

Comment:
Why not just put `_pendingOfferSendResponses--` right here, at the top of the callback?